tests/bulk_create/models.py

```diff
@@ -51,3 +54,20 @@ class TwoFields(models.Model):
 
 class NoFields(models.Model):
     pass
+
+
+class NullableFields(models.Model):
+    """contains all available fields specified in db.backends.oracle.BulkInsertMapper"""
+    big_int_filed = models.BigIntegerField(null=True, default=1)
+    binary_field = models.BinaryField(null=True, default=b'data')
+    date_field = models.DateField(null=True, default=timezone.now)
+    datetime_field = models.DateTimeField(null=True, default=timezone.now)
+    decimal_field = models.DecimalField(null=True, max_digits=2, decimal_places=1, default='1.1')
+    duration_field = models.DurationField(null=True, default=datetime.timedelta(1))
+    float_field = models.FloatField(null=True, default=3.2)
+    integer_field = models.IntegerField(null=True, default=2)
+    null_boolean_field = models.NullBooleanField(null=True, default=False)
+    positive_integer_field = models.PositiveIntegerField(null=True, default=3)
+    positive_small_integer_field = models.PositiveSmallIntegerField(null=True, default=4)
+    small_integer_field = models.SmallIntegerField(null=True, default=5)
+    time_field = models.TimeField(null=True, default=datetime.datetime.now().time())
```

Comment:
This is causing a system check warning:
```
Fixed default value provided.
	HINT: It seems you set a fixed date / time / datetime value as default for this field. This may not be what you want. If you want to have the current date as default, use `django.utils.timezone.now`
```
Could it be fixed?